README.rst

```diff
@@ -136,6 +136,9 @@ or, say, the name used in EvtGen, you can get a particle directly.
     >>>
     >>> Particle.from_evtgen_name("J/psi")
     <Particle: name="J/psi(1S)", pdgid=443, mass=3096.900 ± 0.006 MeV>
+    >>>
+    >>> Particle.from_nucleus_info(a=12, z=6)
```

Comment:
I understand your point about a method special for nuclei. I can see the interest, especially if we want to expand the relevance of the package to other colleagues.

Just note for completeness that you can also do

```
>>> Particle.findall(lambda p: p.pdgid.A == 12 and p.pdgid.Z == 6)
[<Particle: name="C12", pdgid=1000060120, mass=11177.9291399 MeV>]
```

which is as here but a list, though, since using one of the find methods.

Reply:
I didn't even know that PDGID has `.A` and `.Z`, nice!

Reply:
You can see *a lot" in the notebook demo but point taken that we need to invest on a Sphinx doc. The fact that you did not see it should push this up in priority ... Of course any help is welcome :-).